osu.Game.Rulesets.Catch/Objects/CatchHitObject.cs

```diff
@@ -90,7 +90,7 @@ protected override void ApplyDefaultsToSelf(ControlPointInfo controlPointInfo, B
 
             TimePreempt = (float)BeatmapDifficulty.DifficultyRange(difficulty.ApproachRate, 1800, 1200, 450);
 
-            Scale = 1.0f - 0.7f * (difficulty.CircleSize - 5) / 5;
+            Scale = (1.0f - 0.7f * (difficulty.CircleSize - 5) / 5) / 2;
```

Comment:
I'm not sure why this is halved, but my working hypothesis is "it just needs to be".

Reply:
it's now 1:1 with the osu! ruleset implementation, but yeah, "because it needs to be" is about right.